pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright © 2020 ForgeRock AS (obst@forgerock.com)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.forgerock.securebanking</groupId>
+    <artifactId>securebanking-openbanking-uk-rcs</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>securebanking-openbanking-uk-rs</name>
+    <description>A UK Open Banking Remote Consent Service for the Secure Banking Accelerator Toolkit</description>
+
+    <modules>
+        <module>securebanking-openbanking-uk-rcs-api</module>
+        <module>securebanking-openbanking-uk-rcs-server</module>
```

Comment:
Too bad this couldn't be called `securebanking-openbanking-uk-rcs` (unless we change the top level pom). Happy to call it something else.

Reply:
Is it the interfaces module? we can call it `rcs-api` and the implementation `rcs-api-implementation` or something like that.

Reply:
It's the module containing the bulk of the code. Implementation is a possibility, but it's a bit long :-)

I've called it rcs-server, as that's what it's called in openbanking-aspsp (which also uses `-sample` for the app module).